inst/examples/showcase/static/css/styles.css

```
@@ -1,9 +1,67 @@
+html, body, .grid {
+  height: 100%;
+}
+
 body {
   margin: 0;
+  line-height: 1.3rem;
 }
 
 .sidebar {
-  padding-left: 10px;
+  padding-top: 20px;
+}
+
+.home-button {
+  display: flex;
+  align-items: center;
+  font-size: 20px;
+  font-weight: 500;
+  padding-left: 15px;
+  padding-bottom: 20px;
+  border-bottom: 1px solid #ddd;
+}
+
+.home-button::before, .home-button::after {
+  content: "";
+  height: 0;
+  border-top: 1px solid #215db0;
+  opacity: 0.5;
+}
+
+.home-button::before {
+  width: 23%;
+  margin-right: 15px;
+}
+
+.home-button::after {
+  width: 30%;
+  margin-left: 15px;
+}
+
+.home-button:focus {
+  outline: none;
+  text-decoration-line: underline;
+}
+
+.home-page {
+  display: flex;
+  flex-direction: column;
+}
+
+.home-page-title {
+  margin: 5px 0;
+}
+
+.home-page-section {
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  padding: 25px;
+  margin-top: 35px;
+}
+
+.font-mono {
+  font-family: monospace;
 }
 
```

[optional] if you add the new cards, the logo size needs to be smaller than the originals

```suggestion

.logo {
  height: 100px;
}
```